docs/src/tutorials/advanced_simulations.md

```diff
@@ -0,0 +1,282 @@
+```@example ex3
+rn_2 = @reaction_network begin
+    @parameters η
+    (k1,k2), X1 <--> X2
+end
+u0 = [:X1 => 10.0, :X2 => 10.0]
+tspan = (0.0,10.0)
+p_2 = [:k1 => 1.0, :k2 => 1.0, :η => 0.1]
+
+sprob_2 = SDEProblem(rn_2,u0,tspan,p_2; noise_scaling=(@parameters η)[1])
+```
+Here, we first need to add `η` as a parameter to the system using the `@parameters η` option. Next, we pass the `noise_scaling=(@parameters η)[1]` argument to the `SDEProblem`. We can now simulate our system and confirm that noise is reduced:
+```@example ex3
+sol_2 = solve(sprob_2)
+plot(sol_2; idxs=1, ylimit=(0.0,20.0))
+```
+
+Finally, it is possible to set individual noise scaling parameters for each reaction of the system. Our model has two reactions (`X1 --> X2` and `X2 --> X1`) so we will use two noise scaling parameters, `η1` and `η2`. We use the following syntax:
+```@example ex3
+rn_3 = @reaction_network begin
+    @parameters η1 η2
+    (k1,k2), X1 <--> X2
+end
+u0 = [:X1 => 10.0, :X2 => 10.0]
+tspan = (0.0,10.0)
+p_3 = [:k1 => 1.0, :k2 => 1.0, :η1 => 0.1, :η2 => 1.0]
+
+sprob_3 = SDEProblem(rn_3,u0,tspan,p_3; noise_scaling=@parameters η1 η2)
+```
+plotting the results, we see that we have less fluctuation than for the first simulation, but more as compared to the second one (which is as expected):
+```@example ex3
+sol_3 = solve(sprob_3)
+plot(sol_3; idxs=1, ylimit=(0.0,20.0))
+```
+
+
+
+### Useful plotting options
+Catalyst, just like DifferentialEquations, uses the Plots package for all plotting. For a detailed description of differential equation plotting, see [DifferentialEquations documentation on the subject](https://docs.sciml.ai/DiffEqDocs/stable/basics/plot/). Furthermore, the [Plots package documentation](https://docs.juliaplots.org/stable/) contains additional information and describes [a large number of plotting options](https://docs.juliaplots.org/stable/attributes/). Here follows a very short tutorial with a few useful options.
+
+Let us consider the Brusselator model:
+```@example ex4
+using Catalyst, DifferentialEquations, Plots
+
+brusselator = @reaction_network begin
+    A, ∅ → X
+    1, 2X + Y → 3X
+    B, X → Y
+    1, X → ∅
+end
+u0 = [:X => 1.0, :Y => 0.0]
+tspan = (0.0,50.0)
+p = [:A => 1.0, :B => 4.0]
+
+oprob = ODEProblem(brusselator, u0, tspan, p)
+sol = solve(oprob)
+plot(sol)
+```
+If we want to plot only the `X` species, we can use the `idxs` command:
+```@example ex4
+@unpack X = brusselator
+plot(sol; idxs=[X])
+```
+Here we use the `brusselator.X` notation to denote that we wish to plot the `X` species. The input to `idxs` is a vector listing all the species we wish to plot. If we wish to plot a single species, vector notation is not required and we could simply write `plot(sol; idxs=brusselator.X)`. 
+(The plotting feature that automatically sets the label when using this interface is currently not working optimally, hence we manually set the label using the `label` option.)
```

Comment:
You can drop this comment since you can just use `X` now. Also I don't think it is broken, the problem is you didn't name your system. If you gave it a simple name the legend would look fine (i.e. should use name.X). Since you didn't give a name the name that is used is what `gensym` created.

Reply:
Right, fixed this as well.